guvnor-ala/guvnor-ala-wildfly/guvnor-ala-wildfly-provider/src/main/java/org/guvnor/ala/wildfly/executor/WildflyRuntimeExecExecutor.java

```diff
@@ -28,61 +28,70 @@
 import static com.google.common.base.Strings.isNullOrEmpty;
 import static org.guvnor.ala.util.RuntimeConfigHelper.buildRuntimeName;
 
-public class WildflyRuntimeExecExecutor<T extends WildflyRuntimeConfiguration> implements RuntimeBuilder<T, WildflyRuntime>,
-        RuntimeDestroyer,
-        FunctionConfigExecutor<T, WildflyRuntime> {
+public class WildflyRuntimeExecExecutor<T extends WildflyRuntimeConfiguration>
+        implements RuntimeBuilder<T, WildflyRuntime>,
+                   RuntimeDestroyer,
+                   FunctionConfigExecutor<T, WildflyRuntime> {
```

Comment:
This code was left this way by the creator, and there are more places than 2) to change, but let's see if it's possible.